packages/server-wallet/src/logger.ts

```diff
@@ -1,14 +1,23 @@
 import pino from 'pino';
 
 import {defaultConfig, ServerWalletConfig} from './config';
+import {WALLET_VERSION} from './version';
 
 export function createLogger(config: ServerWalletConfig): pino.Logger {
   // eslint-disable-next-line no-process-env
   const destination =
     config.logDestination && config.logDestination.toLocaleLowerCase() !== 'console'
       ? pino.destination(config.logDestination)
       : undefined;
-  return destination ? pino({level: config.logLevel}, destination) : pino({level: config.logLevel});
+  return destination
+    ? pino({level: config.logLevel}, destination).child({
+        dbName: config.postgresDBName,
+        walletVersion: WALLET_VERSION,
+      })
+    : pino({level: config.logLevel}).child({
+        dbName: config.postgresDBName,
+        walletVersion: WALLET_VERSION,
+      });
```

Comment:
```suggestion
  return pino({level: config.logLevel}, destination).child({
        dbName: config.postgresDBName,
        walletVersion: WALLET_VERSION,
      });
```
Shouldn't this work?

Reply:
Yeah I think it should

Reply:
hmm maybe not

Reply:
7049bea